data/moves.js

```diff
@@ -593,7 +593,7 @@ let BattleMovedex = {
 			let success = false;
 			for (const ally of pokemon.side.pokemon) {
 				if (ally !== source && ((ally.hasAbility('sapsipper')) ||
-						(ally.volatiles['substitute'] && !move.infiltrates))) {
+					(ally.volatiles['substitute'] && !move.infiltrates))) {
```

Comment:
I get that we're inconsistent about indentation, but let's not go reindenting everything in unrelated PRs. Feel free to submit a PR with a lint rule to make this sort of thing consistent, though.

Reply:
Ah. That's something my IDE did (Webstorm). It picked up the editor config and "fixed it" for the entire file while I was working on it. I will revert these lines.

Reply:
I guess it's not worth worrying about if it's difficult to configure your IDE not to do this.

Reply:
Probably the next person using Webstorm would be doing the same thing. Luckily it only fixes it in like 3 places or so. I'll keep it in mind for the future then.

Reply:
Not applicable anymore, since adding @urkerab 's change I managed to leave the entire moves.js file alone.